src/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerTrainingService.ts

```diff
@@ -211,16 +211,28 @@ class FrameworkControllerTrainingService extends KubernetesTrainingService imple
             if (this.azureStorageClient === undefined) {
                 throw new Error('azureStorageClient is not initialized');
             }
+            let retryCount: number = 1;
             try {
-                //upload local files, including scripts for running the trial and configuration (e.g., hyperparameters) for the trial, to azure storage
-                await AzureStorageClientUtility.uploadDirectory(
-                    this.azureStorageClient, `nni/${getExperimentId()}/${trialJobId}`, this.azureStorageShare, `${trialLocalTempFolder}`);
-                //upload code files to azure storage
-                await AzureStorageClientUtility.uploadDirectory(
-                    this.azureStorageClient, `nni/${getExperimentId()}/${trialJobId}`, this.azureStorageShare, `${this.fcTrialConfig.codeDir}`);
-
-                trialJobOutputUrl = `https://${this.azureStorageAccountName}.file.core.windows.net/` + 
-                                    `${this.azureStorageShare}/${path.join('nni', getExperimentId(), trialJobId, 'output')}`;
+                while (retryCount >= 0) {
```

Comment:
It's a little confusion here because even retryCount == 0, we will enter the loop. 

I  think change to do...while is better to understand:

retryCount = 1
do
{
} while( retyrCount-- > 0)

Reply:
fixed, changed logic.